bittensor/_axon/axon_impl.py

```diff
@@ -127,6 +127,21 @@ def __str__(self) -> str:
     def __repr__(self) -> str:
         return self.__str__()
 
+    def sign_response( self, response_tensors ) -> str:
+        """ Signs the response tensors with the axon key and returns the signature as a string.       
+            Args:
+                response_tensors (:obj:`List[bittensor.proto.Tensor`, `required`): 
+                    response_tensors to be signed            
+            Returns:
+                signature (str, required): 
+                    The string signature generated by signing the response tensors.
+        """
+        signature_messages = self.wallet.hotkey.ss58_address
+        if response_tensors != None:
+            for resp_tensor in response_tensors:
+                signature_messages += resp_tensor.serializeToString()
```

Comment:
This will lead to a significant increase in network bandwidth, no? It will almost double the amount of bandwidth per message.

Reply:
Would it be better if the signature were the hotkey of the recipient?